.github/codespell_ignore_words.txt

```diff
@@ -0,0 +1,4 @@
+ba
+complies
+ot
+siz
```

Comment:
ba, ot, siz - I can see that they are used as abbreviations (uppercase), but there are a lot more of these. Why these ones? Also: codespell doesn't know "complies"?

Reply:
These ones may be typos (for example `ba ==> be`).

About `complies`, see codespell-project/codespell#2062. IIt is a bug or feature of codespell.

Reply:
If these are typos, wouldn't it be better to fix them?

Reply:
I could add parameter [`builtin`](https://github.com/codespell-project/actions-codespell#parameter-builtin) to the GitHub action configuration:
```
with:
  builtin: clear
```

Reply:
They are not typos. Just false positives.